src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -10171,12 +10171,13 @@ def value_counts(
         ascending: bool = False,
         bins: Optional[int] = None,
         dropna: bool = True,
+        include_index: bool = False,
```

Comment:
@sfc-gh-vbudati since you start adding new interface for some of the query compiler interface now, how about let's do that for all APIs that requires extension to handle index, convert this function into a internal function like _value_counts(xxx). and you can leave the check for multiIndex to the index interface
```
 if include_index:
                if self.is_multiindex():
                    ErrorMessage.not_implemented(
                        "`value_counts` is not yet implemented for MultiIndex objects"
                    )
```